flecs_ecs_derive/src/lib.rs

```diff
@@ -868,7 +868,6 @@ impl Parse for Access {
     }
 }
 
-#[derive(Debug)]
```

Comment:
why was this removed? just curious

Reply:
Some `syn` types like `LitStr` stopped implementing debug so the derive stopped working. These are all types internal to the macro so there's no user impact.